support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouSupportReminder.jsx

```diff
@@ -140,7 +142,9 @@ const ContributionThankYouSupportReminder = ({
     const year = selectedDate.getFullYear();
     const month = selectedDate.getMonth() + 1; // javascript dates run from 0-11, we want 1-12
     const paddedMonth = month.toString().padStart(2, '0');
-    return `${year}-${paddedMonth}-01 00:00:00`;
```

Comment:
This has lead to a bug with the reminders for the us eoy appeal. Reminders set for the last day of the year, actually got set to the 1st of December (same as Giving Tuesday)